grammars/README.md

```diff
@@ -59,9 +59,13 @@ Parentheses `()` can be used to group sequences, which allows for embedding alte
 
 ## Repetition and Optional Symbols
 
-- `*` after a symbol or sequence means that it can be repeated zero or more times.
-- `+` denotes that the symbol or sequence should appear one or more times.
-- `?` makes the preceding symbol or sequence optional.
+- `*` after a symbol or sequence means that it can be repeated zero or more times (equivalent to `{0,}`).
+- `+` denotes that the symbol or sequence should appear one or more times (equivalent to `{1,}`).
+- `?` makes the preceding symbol or sequence optional (equivalent to `{0,1}`).
+- `{m}` repeats the precedent symbol or sequence exactly `m` times
+- `{m,}` repeats the precedent symbol or sequence at least `m` times
+- `{m,n}` repeats the precedent symbol or sequence at betwen `m` and `n` times (included)
+- `{,n}` repeats the precedent symbol or sequence at most `n` times (included)
```

Comment:
Not sure if we care about it, but if I'm reading the parser logic correctly, then it feels like an undocumented behavior is that `{,}` is equivalent to `*` -- is that correct?. I don't think we need to make a note of it necessarily, but I'm more commenting to 360-check that I'm following along successfully with what you've written thus far.

Reply:
Gooood catch... that was unintentional, and since this doesn't seem standard in regular expressions I've removed it + tested against it